test/role.js

```diff
@@ -1,99 +1,98 @@
+//           password: 'holdon0006',
+//         })
+//         .end((err, res) => {
+//           const token = res.body.adminToken;
 
-          chai
-            .request(app)
-            .post('/api/role')
-            .set('authorization', token)
-            .send({ role: 'operator' })
-            .end((error, response) => {
-              response.should.have.status(200);
-              response.body.should.be.a('object');
-              response.body.should.have.property('message');
-              response.body.should.have.property('role');
-              done();
-            });
-        });
-    });
-  });
+//           chai
+//             .request(app)
+//             .post('/api/role')
+//             .set('authorization', token)
+//             .send({ role: 'driver' })
+//             .end((error, response) => {
+//               response.should.have.status(200);
+//               response.body.should.be.a('object');
+//               response.body.should.have.property('message');
+//               response.body.should.have.property('role');
+//               done();
+//             });
+//         });
+//     });
+//   });
 
-  // test PUT route
-  describe('PUT /api/role/:id', () => {
-    it('It should edit an existing role', (done) => {
-      const roleId = 4;
-      chai
-        .request(app)
-        .post('/users/login')
-        .send({
-          email: 'doe@gmail.com',
-          password: 'holdon0006',
-        })
-        .end((err, res) => {
-          const token = res.body.adminToken;
+//   // test PUT route
+//   describe('PUT /api/role/:id', () => {
+//     it('It should edit an existing role', (done) => {
+//       const roleId = 1;
+//       chai
+//         .request(app)
+//         .post('/users/login')
+//         .send({
+//           email: 'doe@gmail.com',
+//           password: 'holdon0006',
+//         })
+//         .end((err, res) => {
+//           const token = res.body.adminToken;
 
-          chai
-            .request(app)
-            .put('/api/role/' + roleId)
-            .set('authorization', token)
-            .send({ role: 'admin' })
-            .end((error, response) => {
-              response.should.have.status(200);
-              response.body.should.be.a('object');
-              response.body.should.have.property('message');
-              if (error) console.log(error);
-              done();
-            });
-        });
-    });
-  });
-});
+//           chai
+//             .request(app)
+//             .put('/api/role/' + roleId)
+//             .set('authorization', token)
+//             .send({ role: 'admin' })
+//             .end((error, response) => {
+//               response.should.have.status(200);
+//               response.body.should.be.a('object');
+//               response.body.should.have.property('message');
+//               if (error) console.log(error);
+//               done();
+//             });
+//         });
+//     });
+//   });
+// });
```

Comment:
You need to remove commented code

Reply:
resolved now it clear.
thanks for your review